cmd/crdvalidator/manifests/00_namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: crdvalidator-system
```

Comment:
I wonder why this webhook is in a separate namespace.  Can it be in the same namespace as other rukpak components

Reply:
I see.  I didn't know the intention.  I just wondered.  Thanks!

Reply:
I think I'd prefer landing with rukpak-system until we have a concrete want/need to move this outside of rukpak but I won't hold this up while we continue iterating.